src/common/enums/notification.ts

```diff
@@ -43,10 +42,24 @@ export enum DB_NOTICE_TYPE {
   payment_received_donation = 'payment_received_donation',
   payment_payout = 'payment_payout',
 
-  // circle
+  // circle owners
   circle_new_subscriber = 'circle_new_subscriber',
+  circle_new_follower = 'circle_new_follower',
+  // circle_new_broadcast = 'circle_new_broadcast', // deprecated
+  circle_new_discussion = 'circle_new_discussion',
   circle_new_unsubscriber = 'circle_new_unsubscriber',
   circle_invitation = 'circle_invitation',
+  circle_member_broadcast = 'circle_member_broadcast',
```

Comment:
TBC: L52-L55, did not match with design?

Reply:
![image](https://user-images.githubusercontent.com/4065233/183236359-a82a64b0-ff72-445b-badd-48ae6e963243.png)